### PR TITLE
[W-11689882] build warnings 2.1

### PR DESCRIPTION
**Pull request by cheungaryk**
ref: W-11689882

fix some things to reduce build warnings.

# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [x] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/dw-core-functions-mapobject.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ output application/xml
 ----
 
 [[output]]
-==== Output
+=== Output
```
Comment (dukesphere) [inline modules/ROOT/pages/dw-core-functions-mapobject.adoc:58]:
```suggestion
==== Output
```
restoring H4

**Review by dukesphere — COMMENTED** [review]
@cheungaryk can you approve this commit suggestion restoring the H4?

File: modules/ROOT/pages/dw-core-functions-mapobject.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (dukesphere) [reply modules/ROOT/pages/dw-core-functions-mapobject.adoc:58]: And @cheungaryk can you change the Input one to H4 "==== Input" ?

[the remark this answers is not in the document]
Comment (dukesphere) [reply modules/ROOT/pages/dw-core-functions-mapobject.adoc:58]: In other words, should look like this for Input and Output:

```
[[input_mapobject]]
==== Input

[source,XML,linenums]
----
<?xml version='1.0' encoding='UTF-8'?>
<prices>
    <basic>9.99</basic>
    <premium>53</premium>
    <vip>398.99</vip>
</prices>
----

[[output]]
==== Output
```

**Review by dukesphere — COMMENTED** [review]
see comment

**Comment by cheungaryk** [comment]
> @cheungaryk can you approve this commit suggestion restoring the H4?

Nice catch. Done!

**Review by dukesphere — APPROVED** [review]

**Pull request merged**